cheddar/cheddar-integration-aws/src/test/java/com/clicktravel/infrastructure/persistence/aws/dynamodb/DynamoDbTemplateTest.java

```diff
@@ -1927,4 +1930,182 @@ public void shouldNotBatchWriteItems_withListOfItemsAndFailedIndexCreation() thr
 
     }
 
+    @Test
+    public void shouldFetchEmptyList_withAttributeQueryWithEmptyAttributeList() throws Exception {
+        // Given
+        final AttributeQuery query = mock(AttributeQuery.class);
+        final Condition mockCondition = mock(Condition.class);
+        when(mockCondition.containsNonNullOrEmptyValues()).thenReturn(false);
```

Comment:
As above but the opposite is true here - we are testing that when the condition doesn't contain an comparison value we don't call Dynamo.